app/services/email_sender_service/email_sender.rb

```diff
@@ -0,0 +1,45 @@
+module EmailSenderService
+  class EmailSender
+    def call(address:, subject:, body:)
+      provider.call(
+        address: email_or_email_override(address),
+        subject: subject,
+        body: body
+      )
+    end
+
+  private
+
+    def config
+      EmailAlertAPI.config.email_service
+    end
+
+    def email_override
+      config.fetch(:email_address_override)
```

Comment:
I don't know if it makes sense to use `.fetch` here, an absence of this key can be taken to mean they don't want an override.

Reply:
changed in 4a23d1cfb76349031d30739fe883ede788b6beb0